src/nextTick.ts

```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue'
+import { currentVue } from './runtimeContext'
+
+type NextTick = Vue['$nextTick']
+
+export const nextTick: NextTick = function nextTick(
+  this: ThisType<NextTick>,
+  ...args: Parameters<NextTick>
+) {
+  return currentVue?.nextTick.bind(this, args)
```

Comment:
if called outside of a component setup/lifecycle `currentVue` won't have a value here, we should `Vue.nextTick` in that case

Reply:
Yeah currentVue is Vue and it should always exist after installing, I guess currentVM might be what you are talking.

Reply:
That's right, read as `currentVM` 😅